open-problems.md

```diff
@@ -0,0 +1,5 @@
+# Open problems
+
+This page is a stub. It will be filled out with open problems.
```

Comment:
```suggestion
This page is a stub. It will be filled out with open research problems.
```

Reply:
not sure it makes a lot of sense to update this page considering https://github.com/vacp2p/vac.dev/issues/81 but I updated this